inc/composer/class-plugin.php

```diff
@@ -70,4 +73,37 @@ public function install_files() {
 			mkdir( $dest . '/content/themes' );
 		}
 	}
+
+	/**
+	 * Generate the manifest of module entrypoints to be included automatically.
+	 */
+	public function generate_module_manifest() {
+		$repository = $this->composer->getRepositoryManager()->getLocalRepository();
+		$packages = $repository->getCanonicalPackages();
+		$vendor_dir = $this->composer->getConfig()->get( 'vendor-dir' );
+		$package_list = array_reduce( $packages, function ( $carry, PackageInterface $package ) use ( $vendor_dir ) {
```

Comment:
Could we avoid `array_reduce` here and just `foreach` instead, then `implode` later? 😬 I think it would read clearer, plus `continue` would be clearer than `return $carry`